ARrehab/ARrehab/Minigames/ExpressionGame/ExpressionGame.swift

```diff
@@ -10,8 +10,12 @@ import Foundation
 import ARKit
 import RealityKit
 
+/**
+ Expression Minigame.
+ Prompts you to make various facial expressions.
+ */
 class ExpressionGame: Minigame {
-    
+    // Lets put requiresSession into the overall Minigame class.
```

Comment:
@sanathsengupta was there a particular purpose for requiresSession other than notifying the reader that it requires a Session?

Reply:
Sorry, that was actually left over from an earlier implementation — not necessary anymore, but could still be useful to have down the line